CHANGELOG.md

```diff
@@ -2,6 +2,38 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.2.0](https://github.com/rhobs/monitoring-stack-operator/commit/) (2025-05-19)
```

Comment:
This is the auto generated one, isn't it? Not sure why it skipped 1.1.0, so we'll need to adjust it manually later probably

Reply:
right